modules/nw-overlapped-sharding.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-cluster-traffic-ingress-controller.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-overlapped-sharding_{context}"]
+== Overlapped sharding example
+
+In addition to `finops-router` and `dev-router` in the previous example, you also have `devops-router`, which is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `dev` and `ops`:
```

Comment:
Better change to:
In addition to `finops-router` and `dev-router` in the previous example, you also have `devops-router`, which is configured with the label selector "spec.namespaceSelector.matchExpressions" with key name and values in [dev, ops]

Reply:
:heavy_check_mark: